### ISSUE TITLE
Delete curve

### ISSUE THREAD

**Issue by PetroDude** [kinverarity1/lasio#155]
A delete curve method(or really any non-required section item) would be very useful).

**Comment by dagrha** [comment]
Until this feature is added, you can do something like this to delete a curve from your LASFile object (correct me if I'm wrong, @kinverarity1):

```
def delete_curve(las_object, mnemonic):
    index = las_object.curves.keys().index(mnemonic)
    las_object.curves.pop(index)
    las_object.data = np.delete(las_object.data, np.s_[index], axis=1)
```

**Comment by kinverarity1** [comment]
This has been partly fixed with @dagrha's PR #156. To delete a curve use:

lasfile_obj.delete_curve(mnemonic)

Deleting a section item is yet to be done so I will leave the issue open for now
